tasks/templates/compare-listing.njk

```diff
@@ -8,7 +8,7 @@
 
         <!-- Load all component stylesheets -->
         {% for component in allComponents -%}
-            {% if component not in ["actionmenu", "commons", "overlay"] %}
```

Comment:
ActionMenu is the funny one that isn't _really_ a component. Is it removed here because we're planning to no longer keep it in the build?